# This file is distributed under the same license as the Django package.
#
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-08-13 16:08+0100\n"
"PO-Revision-Date: 2010-05-13 15:35+0200\n"
"Last-Translator: Django team\n"
"Language-Team: English <en@li.org>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:49
msgid "Personal info"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:51
msgid "Permissions"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:62
msgid "Important dates"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:158
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:177
msgid "Conflicting form data submitted. Please try again."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:185
msgid "Password changed successfully."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:187
msgid "Password-based authentication was disabled."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:208
#, python-format
msgid "Change password: %s"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/admin.py:210
#, python-format
msgid "Set password: %s"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/apps.py:16
msgid "Authentication and Authorization"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/base_user.py:41
msgid "password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/base_user.py:42
msgid "last login"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:48
msgid "Invalid password format or unknown hashing algorithm."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:56
msgid "No password set."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:59
msgid "Reset password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:59
msgid "Set password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:104
msgid "The two password fields didn’t match."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:107
msgid ""
"Whether the user will be able to authenticate using a password or not. If "
"disabled, they may still be able to authenticate using other backends, such "
"as Single Sign-On or LDAP."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:113
#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:258
#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:294
msgid "Password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:113
msgid "Password confirmation"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:126
msgid "Enter the same password as before, for verification."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:133
msgid "Password-based authentication"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:136
msgid "Enabled"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:136
msgid "Disabled"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:260
msgid ""
"Raw passwords are not stored, so there is no way to see the user’s password."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:276
msgid ""
"Enable password-based authentication for this user by setting a password."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:301
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:304
msgid "This account is inactive."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:369
msgid "Email"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:472
msgid "New password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:472
msgid "New password confirmation"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:497
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/forms.py:501
msgid "Old password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:349
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:442
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:532
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:627
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:678
msgid "algorithm"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:350
msgid "iterations"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:351
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:448
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:534
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:631
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:679
msgid "salt"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:352
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:449
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:632
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:680
msgid "hash"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:443
msgid "variety"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:444
msgid "version"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:445
msgid "memory cost"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:446
msgid "time cost"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:447
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:630
msgid "parallelism"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:533
#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:628
msgid "work factor"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:535
msgid "checksum"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/hashers.py:629
msgid "block size"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:63
#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:117
msgid "name"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:67
msgid "content type"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:69
msgid "codename"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:74
msgid "permission"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:75
#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:120
msgid "permissions"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:127
msgid "group"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:128
#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:259
msgid "groups"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:250
msgid "superuser status"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:253
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:262
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:270
msgid "user permissions"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:272
msgid "Specific permissions for this user."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:346
msgid "username"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:350
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:354
msgid "A user with that username already exists."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:357
msgid "first name"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:358
msgid "last name"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:359
msgid "email address"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:361
msgid "staff status"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:363
msgid "Designates whether the user can log into this admin site."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:366
msgid "active"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:369
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:373
msgid "date joined"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:382
msgid "user"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/models.py:383
msgid "users"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:111
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
msgstr[1] ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:123
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""
msgstr[1] ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:206
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:213
msgid "Your password can’t be too similar to your other personal information."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:245
msgid "This password is too common."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:250
msgid "Your password can’t be a commonly used password."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:261
msgid "This password is entirely numeric."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/password_validation.py:266
msgid "Your password can’t be entirely numeric."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only unaccented lowercase a-z "
"and uppercase A-Z letters, numbers, and @/./+/-/_ characters."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:164
msgid "Logged out"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:224
msgid "Password reset"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:252
msgid "Password reset sent"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:263
msgid "Enter new password"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:335
msgid "Password reset unsuccessful"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:345
msgid "Password reset complete"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:357
msgid "Password change"
msgstr ""

#: myenv/lib/python3.12/site-packages/django/contrib/auth/views.py:380
msgid "Password change successful"
msgstr ""
